nss-cohort-37/Bangazon-Site-Green-Monkeys
# Et migration

# Description
Ticket # 17 – Part 1) Added the ability to View and Edit User Profile

View these items 
•	First Name
•	Last Name
•	Address
•	Phone Number
•	Payment Options – link not completed
•	Order History – link is working

Added ability to EDIT Profile Items 
•	Address
•	Phone
•	Last Name

## Type of change
- [x ] New feature (non-breaking change which adds functionality)

# Testing Instructions

Note - Login into the system as Admin@admin.com and use Password Admin8* 

Go to Account View and click on the userid (aka email) near the upper right navigation bar area and review the items on the Profile page to confirm you can see

•	First Name
•	Last Name
•	Address
•	Phone Number
•	Payment Options – link not completed
•	Order History

To test the EDIT of the fields try entering new information into the pre-populated form fields to edit their address, phone number, and last name.

Press Save button.   The form should now retain and render any changes you made.  

You can logout and return to Profile to confirm changes OR you can simply leave the page view using another link like Email or Password and then return to the Profile view by clicking on the Profile link button. 

Ticket # 17 – Part 2) Added the ability to View User Order History

View these items 

•	Order Number
•	Order Date
•	Payment Type
•	Order Total Amount

## Type of change
- [x ] New feature (non-breaking change which adds functionality)

Note - Login into the system as Admin@admin.com and use Password Admin8* 
To test the Order History View click on the Order History link in the upper right navigation bar area.  (You must be logged in to see this link)

Confirm you can see

•	Order Number
•	Order Date
•	Payment Type
•	Order Total Amount

# Checklist:

- [x ] My code follows the style guidelines of this project
- [x ] I have performed a self-review of my own code
- [x ] I have commented my code, particularly in hard-to-understand areas
- [x ] My changes generate no new warnings or errors
- [x ] I have added test instructions that prove my fix is effective or that my feature works
